recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -358,9 +364,19 @@ def requirements(self):
         if self.settings.os != "Windows":
             self.requires("openssl/[>=1.1 <4]")
             self.requires("libcurl/[>=7.78.0 <9]")
-        if self.settings.os in ["Linux", "FreeBSD"]:
+        if self.settings.os in ["Linux", "FreeBSD", "Android"]:
```

Comment:
```suggestion
        if self.settings.os in ["Linux", "FreeBSD"]:
```

Pulseaudio has transitive dependencies no supported for Android.